billboardd/daemon.h

```diff
@@ -282,7 +288,9 @@ class Daemon : public QObject {
         }
 
         void contextChanged() {
-            updateTimer.start();
+            if (tklock.isLocked()) {
```

Comment:
hmm, it looks like it ought to just call render. it doesnt break the preview it for me